### PR TITLE
BIOWDL-420: Replace mv command with cp.

### PR DESCRIPTION
**Pull request by JasperBoom**
In order to be able to use the moved files later in the pipeline, they also need to be present in the working directory.

### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] `parameter_meta` for each task is up to date.

### CODE REVIEW

**Comment by JasperBoom** [comment]
I will add the time_minutes with a separate PR later, so it doesn't conflict with my tests of the overall PacBio pipeline.

File: lima.wdl [hunk 1]
```diff
@@ -87,13 +87,13 @@ task Lima {
         ~{barcodeFile} \
         ~{basename(outputPrefix) + ".fl.bam"}
```
Comment (rhpvorderman) [inline lima.wdl:88]: Can't you use {outputPrefix} here directly instead of the basename ? Wouldn't that remove the necessity for copying?

Comment (JasperBoom) [reply lima.wdl:88]: That would indeed be nicer. But for some reason the glob function in the outputs doesn't work when I try to glob(outputPrefix + "*.bam"). It doesn't find any files. So I need all files to be written to the working dir in order to use them downstream.

Comment (JasperBoom) [reply lima.wdl:88]: The lima task gets one BAM file as input. And can output multiple de-multiplexed BAM files. So that's why I am using glob here.

**Pull request merged**